README.md

```diff
@@ -241,8 +245,22 @@ unityTransport.SetRelayServerData(new RelayServerData(allocation, "wss"));
 unityTransport.UseWebSockets = true;
 ```
 
+**Note** that if using Unity Transport Package 2.4.0, the constructor for RelayServerData(Allocation alloc, string connectionType) does not exist anymore. Hence use below as a example to set the WSS connection type for WebGL builds.
+
+```csharp
+Allocation allocation = await RelayService.Instance.CreateAllocationAsync(maxPlayers);
+var unityTransport = _networkManager.TransortManager.GetTransport<FishyUnityTransport>();
+
+RelayServerData relayServerData = allocation.ToRelayServerData("wss"); //Note that it is important to use lowercase because the ToRelayServerData makes a comparison directly by lowercase.
+unityTransport.UseWebSockets = true;
+unityTransport.SetRelayServerData(relayServerData);
```

Comment:
Should we correct the example instead and write a note for the outdated version? I feel like devs should read about the most up-to-date code first.